apps/server/src/modules/tool/controller/mapper/external-tool-request.mapper.ts

```diff
@@ -6,23 +6,22 @@ import {
 	Lti11ToolConfigDO,
 	Oauth2ToolConfigDO,
 } from '@shared/domain/domainobject/external-tool';
-import { ProviderOauthClient } from '@shared/infra/oauth-provider/dto';
 import { CustomParameterLocation, CustomParameterScope, CustomParameterType, SortOrderMap } from '@shared/domain';
-import { CustomParameterScopeParams } from '../interface/custom-parameter-scope.enum';
-import { CustomParameterLocationParams } from '../interface/custom-parameter-location.enum';
-import { CustomParameterTypeParams } from '../interface/custom-parameter-type.enum';
-import { CustomParameterCreateParams } from '../controller/dto/request/custom-parameter.params';
-import { BasicToolConfigParams } from '../controller/dto/request/basic-tool-config.params';
-import { Oauth2ToolConfigParams } from '../controller/dto/request/oauth2-tool-config.params';
-import { ExternalToolParams } from '../controller/dto/request/external-tool-create.params';
-import { Lti11ToolConfigParams } from '../controller/dto/request/lti11-tool-config.params';
-import { TokenEndpointAuthMethod } from '../interface/token-endpoint-auth-method.enum';
-import { SortExternalToolParams } from '../controller/dto/request/external-tool-sort.params';
-import { ExternalToolSearchParams } from '../controller/dto/request/external-tool-search.params';
+import { CustomParameterScopeParams } from '../../interface/custom-parameter-scope.enum';
+import { CustomParameterLocationParams } from '../../interface/custom-parameter-location.enum';
+import { CustomParameterTypeParams } from '../../interface/custom-parameter-type.enum';
+import { CustomParameterCreateParams } from '../dto/request/custom-parameter.params';
+import { BasicToolConfigParams } from '../dto/request/basic-tool-config.params';
+import { Oauth2ToolConfigParams } from '../dto/request/oauth2-tool-config.params';
+import { ExternalToolParams } from '../dto/request/external-tool-create.params';
+import { Lti11ToolConfigParams } from '../dto/request/lti11-tool-config.params';
+import { SortExternalToolParams } from '../dto/request/external-tool-sort.params';
+import { ExternalToolSearchParams } from '../dto/request/external-tool-search.params';
```

Comment:
Barrel files? So that we can change the import to e.g. 
`{ CustomParameterScopeParams, CustomParameterLocationParams, /* ... */ } from '../dto';`